<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>319</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>306</width>
    <height>319</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f5f5f5;
color: rgb(0, 0, 0);
</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
min-width: 65;
min-height: 40;
max-width: 65;
max-height: 40;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="displayPanel">
      <property name="minimumSize">
       <size>
        <width>290</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>20</pointsize>
        <italic>false</italic>
        <bold>false</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1 solid #b8b8b8;
background-color: #fff;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="actionClear">
        <property name="styleSheet">
         <string notr="true">background-color: #cb1a21;
color: #ffffff;</string>
        </property>
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="actionDel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>42</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>42</height>
         </size>
        </property>
        <property name="text">
         <string>Del</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="actionDiv">
        <property name="text">
         <string>/</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">lockGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="switch1">
        <property name="text">
         <string>(</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">switchGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="num7">
        <property name="text">
         <string>7</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">lockGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="num8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>8</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">lockGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="num9">
        <property name="text">
         <string>9</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">lockGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="actionMul">
        <property name="text">
         <string>x</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">lockGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="switch2">
        <property name="text">
         <string>)</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">switchGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="num4">
        <property name="text">
         <string>4</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">lockGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="num5">
        <property name="text">
         <string>5</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">lockGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="num6">
        <property name="text">
         <string>6</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">lockGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="actionMinus">
        <property name="text">
         <string>-</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">lockGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="switch3">
        <property name="text">
         <string>sqrt</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">switchGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="num1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>67</width>
          <height>42</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>67</width>
          <height>42</height>
         </size>
        </property>
        <property name="text">
         <string>1</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">lockGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="num2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>2</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">lockGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="num3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>67</width>
          <height>42</height>
         </size>
        </property>
        <property name="text">
         <string>3</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">lockGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="actionPlus">
        <property name="text">
         <string>+</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">lockGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="switch4">
        <property name="text">
         <string>pow</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">switchGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="num0">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>42</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>42</height>
         </size>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">lockGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="comma">
        <property name="text">
         <string>.</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">lockGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="switchPage">
        <property name="text">
         <string>Page</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="actionCalc">
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #3490cd;
color: #ffffff;</string>
        </property>
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="switchGroup"/>
  <buttongroup name="lockGroup"/>
 </buttongroups>
</ui>
